### PR TITLE
Refs #10838 - Adds a backend Status widget to dashboard

### PR DESCRIPTION
**Pull request by dustints**
* moves about templates so they can be found in rail's template search path
* register the status template as a default template on init

### CODE REVIEW

**Comment by ehelms** [comment]
May want to update commit message to indicate this is being added to the about page (and not dashboard).

**Comment by ehelms** [comment]
There is a `.swm` file present you probably didn't mean to commit.

**Comment by dustints** [comment]
@ehelms remove the swap file...
This adds the widget to the dashboard and also keeps the existing widget on the about page.

**Comment by ehelms** [comment]
What do we gain by leaving it on the about page?

**Comment by dustints** [comment]
@ehelms I see what you mean; no reason to keep it on that page if the dashboard is the more appropriate place to place the widget.

**Comment by ehelms** [comment]
This PR appears to be adding a new dashboard widget to the main dashboard (and not Katello's current 'Content Dashboard'). Would it make more sense then to use the dashboard plugin API for this? (http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Adding-a-dashboard-widget)

**Comment by thomasmckay** [comment]
@ehelms - This PR appears to have been updated per your comments, is it ready for ack?
@dustints - Thanks for all the hard work!

**Comment by ehelms** [comment]
I will re-review and test it and let you know.

File: app/views/dashboard/_system_status.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h4 class="header" ><%=_("Backend System Status")%></h4>
+<table class="table table-striped">
+  <tr>
+    <th><%= _("Component") %></th>
+    <th><%= _("Status") %></th>
+    <th><%= _("Message") %></th>
+  </tr>
+  <% debugger %>
```
Comment (ehelms) [inline app/views/dashboard/_system_status.html.erb]: debugger

File: app/views/dashboard/_system_status.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<h4 class="header" ><%=_("Backend System Status")%></h4>
+<table class="table table-striped">
+  <tr>
```
Comment (ehelms) [inline app/views/dashboard/_system_status.html.erb]: Can we wrap this in a thead?

File: app/views/dashboard/_system_status.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <th><%= _("Component") %></th>
+    <th><%= _("Status") %></th>
+    <th><%= _("Message") %></th>
+  </tr>
+  <% debugger %>
+  <% Katello::Ping.ping[:services].sort.each do |component, status| %>
+    <tr>
```
Comment (ehelms) [inline app/views/dashboard/_system_status.html.erb]: Can we wrap this in a tbody?

**Comment by ehelms** [comment]
This looks off:

screenshot from 2015-07-07 15 28 06

File: app/views/foreman/overrides/about/_widget_wrapper.html.erb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<div class="<%= classes %>">
```
Comment (ehelms) [inline app/views/foreman/overrides/about/_widget_wrapper.html.erb]: Not sure I see where this file is being used.

**Comment by thomasmckay** [comment]
I'll be taking this PR over; will close and reference new one soon. Thanks @dustints !

**Comment by thomasmckay** [comment]
closing in favor of https://github.com/Katello/katello/pull/5364

**Pull request closed without merging**